### PR TITLE
feat: tictactoe UI

### LINKED ISSUE #110: TicTacToe user interface

### PR DESCRIPTION
**Pull request by matt-user**
## Type of change

- New feature

## Changes

The following changes have been made:

- Create a ui for the tic tac toe app

## Related Issues

Closes #110

### CODE REVIEW

File: games/TicTacToe/README.md
Comment (K1-R1) [inline games/TicTacToe/README.md:57]: Could we reduce the necessary commands to just install and spin up, or maybe even just a single command, via `package.json` scripts?
I've used something like the below before:
```json
"scripts": {
		"build:resources": "tsx generatePredicateResources.ts",
		"build": "run-s build:resources && tsc && vite build",
```

**Review by K1-R1 — COMMENTED** [review]
Looking great; a few minor comments that I'll leave to author's choice. Happy to approve once addressed

File: games/TicTacToe/packages/app/src/App.tsx
Comment (sarahschwartz) [inline games/TicTacToe/packages/app/src/App.tsx:29]: The gameState was undefined for me when I first connected so this was blank, idk if I didn't set it up right?

File: games/TicTacToe/README.md
Comment (sarahschwartz) [inline games/TicTacToe/README.md:63]: Seems like you also need to modify the chainConfig to fund your wallet locally, right?

It would be nice to explain how to play a bit more, and that you have two accounts and connect them both

**Review by sarahschwartz — COMMENTED** [review]
Nice! I just have a couple comments

File: games/TicTacToe/packages/app/src/App.tsx
Comment (matt-user) [reply games/TicTacToe/packages/app/src/App.tsx]: I'm pretty sure this happened bc your wallet was on the wrong network.  I'll come up with some sort of check to make sure the user is on the right network and if not give a warning/error

**Review by K1-R1 — COMMENTED** [review]
When trying the Vercel preview I'm getting `Error: it is not your turn. Please switch to the correct player.` on both accounts.

**Comment by sarahschwartz** [comment]
> When trying the Vercel preview I'm getting `Error: it is not your turn. Please switch to the correct player.` on both accounts.

same

**Comment by matt-user** [comment]
@K1-R1 @sarahschwartz I think I fixed all the ui errors.  After testing the only error I have been encountering is the one discussed with the ts sdk team and at the moment I can't fix that on my end.

**Review by K1-R1 — APPROVED** [review]
Awesome, it's working great now! 
Would you mind linking the TS-SDK issue on this PR? Do you think we should wait for the TS-SDK fix before merging?
And just as a nice addition in the meantime; is it possible to prompt the account switch between turns, or add a switch account button?

**Review by sarahschwartz — APPROVED** [review]

**Comment by matt-user** [comment]
> Awesome, it's working great now! Would you mind linking the TS-SDK issue on this PR? Do you think we should wait for the TS-SDK fix before merging? And just as a nice addition in the meantime; is it possible to prompt the account switch between turns, or add a switch account button?

I don't think an issue was created for this bug, but the sdk team said they are looking into it. Looking at the wallet sdk I don't see a way to add a button to switch accounts.

**Pull request merged**
